shell/mixins/__tests__/chart-product-redirect.test.ts

```diff
@@ -0,0 +1,93 @@
+import { shallowMount } from '@vue/test-utils';
+import ChartProductRedirect from '@shell/mixins/chart-product-redirect';
+
+describe('chart Product Redirect', () => {
+  const PRODUCT = 'PRODUCT';
+  const CHART_NAME = 'CHART_NAME';
+
+  const createComponent = (...args: Parameters<typeof ChartProductRedirect>) => ({
+    props:  {},
+    render() { },
+    mixins: [ChartProductRedirect(...args)]
+  });
+
+  it('should exit early because the product is already active', async() => {
```

Comment:
So, this test is mostly about coverage and it's verifying that we exit beforeCreate here https://github.com/rancher/dashboard/pull/10971/files#diff-902296d80668085dc0fe9c31e2d89c2ea9c00e5d9bdc08b479b83749eb5dbfdfR11-R13

I'm verifying that we don't execute any of the code below by checking to make sure the currentCluster mock getter is never called which implies we've exited early.

Reply:
Yes, I've seen it later in the code. I was just saying that it was not clear by just reading the test and I wanted just to be sure of other possible cases. 
Otherwise it's fine like this 👍